public/video-ui/src/components/VideoEdit/formComponents/PrivacyStatus.js

```diff
@@ -19,7 +19,7 @@ export default class PrivacyStatusSelect extends React.Component {
     return (
       <SelectBox
         fieldName="Privacy Status"
-        fieldValue={this.props.video.privacyStatus}
+        fieldValue={this.props.video ? this.props.video.privacyStatus : ''}
```

Comment:
Its only there for when the component is still loading, I don't know if it makes sense to say that a video that hasn't loaded yet is private but I don't think it would make much of a difference in any case.